pages/glossary/[id].tsx

```diff
@@ -39,10 +40,10 @@ export async function getStaticPaths() {
     traverseCollections: false,
   })
 
-  const paths = Object.keys(pages)
-    .map((pageId) => mapPageUrl(pageId))
-    .filter((path) => path && path !== "/")
-    .map((path) => `/glossary${path}`)
+  const paths = Object.entries(pages)
+    .map(([key, page]) => (page && key ? [mapPageUrl(key), getPageTitle(page)] : ["", ""]))
+    .filter(([id, title]) => id && title && title !== "Glossary")
+    .map(([id, path]) => `/glossary/${path.toLowerCase().replaceAll(" ", "-")}-${id.replaceAll("/", "")}`)
```

Comment:
This is looking too complex.

Reply:
I might extract logic into some util function to share it.